src/styles/sharedStyles.tsx

```diff
@@ -199,21 +199,44 @@ export const readerFundedSubscribeCardHolderCss = css`
     flex-wrap: wrap;
     justify-content: space-between;
     & > * {
-      width: 33.3%;
-      width: calc(33.3% - 13px);
+      width: 50%;
```

Comment:
Sorry it's all can find :) Doesn't the second width below make this one redundant?

Reply:
Yeah this one should be removed